src/pages/admin/waitlist.tsx

```diff
@@ -12,6 +12,7 @@ import Head from "next/head";
 import { useRouter } from "next/router";
 import { getCookie } from "<@>/utils/cookie";
 
+
 const WaitList: React.FC = () => {
   const [params, setParams] = useState({
     pageNumber: 1,
```

Comment:
add error handling for this component: for instance: useApplicantWaitlistQuery

Reply:
> add error handling for this component: for instance: useApplicantWaitlistQuery

Can you elaborate this comment? I used error handler on line 130.